go/runtime/bundle/registry.go

```diff
@@ -152,6 +152,37 @@ func (r *Registry) AddManifest(manifest *Manifest, dir string) error {
 	return nil
 }
 
+// Manifests returns all known manifests (regular and detached).
```

Comment:
```suggestion
// Manifests returns all known manifests.
```
In the future, attached ROFL components might be referred to as system ROFL components. To avoid needing to revise the comments later, I recommend keeping this simple.